README.md

```diff
@@ -33,20 +33,21 @@ customer = planship.create_customer({
     "email:": "vader@empire.gov"
 })
 
-# Subscribe the customer with a given ID to a plan with a slug "medium"
+# Subscribe the customer to a plan with the slug "medium"
 subscription = planship.create_subscription(customer.id, "medium")
 
-# Retrieve entitlements for a customer with a give ID
+# Retrieve entitlements for the customer
 entitlements = planship.get_entitlements(customer.id)
 
-# Report usage for a customer with given ID for a given metering ID
+# Report 11 units of usage for the "api-call" metering ID for the customer
 planship.report_usage(customer.id, "api-call", 11)
 ```
 
-The complete reference for the `Planship` class can be found [here](./docs/content/planship-class.md).
+The complete reference for the `Planship` class can be found [here](https://github.com/planship/planship-python/blob/master/docs/content/planship-class.md.
 
 ## Links
 
 - [Planship documentation](https://docs.planship.io)
```

Comment:
Minor nit but maybe rename "Planship console sign-up" to "Planship sign-up" and move it to just above or below docs so general Planship links are grouped separately from the Python stuff.